_posts/2019-09-16-picture-archivist-in-f.html

```diff
@@ -484,7 +484,7 @@ <h3 id="f30093164b184bbf877f307fa4cf4c63">
 		First, you load the <code>sourceTree</code> using the <code>readTree</code> operation. This returns a <code>Tree&lt;string,string&gt;</code>, so map the leaves to <code>FileInfo</code> objects. You then load the image metatadata by traversing <code>sourceTree</code> with <code>Tree.choose readPhoto</code>. Each call to <code>readPhoto</code> produces a <code>PhotoFile option</code>, so this is where you want to use <code>Tree.choose</code> to throw all the <code>None</code> values away.
 	</p>
 	<p>
-		Those two lines of code is the initial impure step of the sandwich (yes: mixed metaphors, I know).
+		Those two lines of code are the initial impure step of the sandwich (yes: mixed metaphors, I know).
```

Comment:
This is a problematic sentence, I admit. There are two lines of code (plural), but one step (singular). How about "Those two lines of code constitute the initial impure step"?

Reply:
Yes, that works.  I just force pushed that change.